packages/uikit/src/components/Card/CardRibbon.tsx

```diff
@@ -7,7 +7,7 @@ interface StyledCardRibbonProps extends CardRibbonProps {
 }
 
 const StyledCardRibbon = styled.div<Partial<StyledCardRibbonProps>>`
-  z-index: 10;
+  z-index: ${({ theme }) => theme.zIndices.ribbon};
```

Comment:
z index was 10 now it is set to 9, is this intended?

Reply:
Cause, the zIndex of a Ribbon should be smaller than that of a drodown.
The zIndex of dropdown is set to 10

Reply:
👍